source/blog/2018-09-28-the-ember-times-issue-66.md

```diff
@@ -17,7 +17,12 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [The Addon Test Is Right Where It Belongs ❤️](https://github.com/emberjs/rfcs/pull/378)
+
+<!--alex ignore host-hostess-->
+Spending too much time looking for the **test files** of your **in-repo addon**? Feeling exhausted to maintain a large test suite in which addon and host app test cases are meshed together? It doesn't have to be this way.
```

Comment:
Minor nit "Feeling exhausted to maintain a large test suite" --> "Feeling too exhausted to maintain a large test suite"